src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,160 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final String OUT_OF_BOUNDS_EXCEPTION = "Index out of bounds!";
+    private int size;
+    private Node<T> first;
+    private Node<T> last;
+
+    private static class Node<T> {
+        private T item;
+        private Node<T> next;
+        private Node<T> prev;
+
+        Node(Node<T> prev, T element, Node<T> next) {
+            this.item = element;
+            this.next = next;
+            this.prev = prev;
+        }
+    }
+
+    public MyLinkedList() {
+        this.size = 0;
+    }
```

Comment:
For type int, the default value is zero. We also don't really need a empty constructor if we remove size from it.
```suggestion 
```